src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/config/EmailConfiguration.kt

```diff
@@ -113,4 +146,43 @@ data class EmailTemplates(
   val probationBookingRequestUser: String,
   val probationBookingRequestPrisonProbationTeamEmail: String,
   val probationBookingRequestPrisonNoProbationTeamEmail: String,
-)
+  val transferCourtBookingCourt: String,
+  val transferCourtBookingPrisonCourtEmail: String,
+  val transferCourtBookingPrisonNoCourtEmail: String,
+  val releaseCourtBookingCourt: String,
+  val releaseCourtBookingPrisonCourtEmail: String,
+  val releaseCourtBookingPrisonNoCourtEmail: String,
+) {
+
+  private val emailTemplateMappings = mapOf(
+    NewCourtBookingUserEmail::class.java to newCourtBookingUser,
+    NewCourtBookingPrisonCourtEmail::class.java to newCourtBookingPrisonCourtEmail,
+    NewCourtBookingPrisonNoCourtEmail::class.java to newCourtBookingPrisonNoCourtEmail,
+    AmendedCourtBookingUserEmail::class.java to amendedCourtBookingUser,
+    AmendedCourtBookingPrisonCourtEmail::class.java to amendedCourtBookingPrisonCourtEmail,
+    AmendedCourtBookingPrisonNoCourtEmail::class.java to amendedCourtBookingPrisonNoCourtEmail,
+    CancelledCourtBookingUserEmail::class.java to cancelledCourtBookingUser,
+    CancelledCourtBookingPrisonCourtEmail::class.java to cancelledCourtBookingPrisonCourtEmail,
+    CancelledCourtBookingPrisonNoCourtEmail::class.java to cancelledCourtBookingPrisonNoCourtEmail,
+    CourtBookingRequestUserEmail::class.java to courtBookingRequestUser,
+    CourtBookingRequestPrisonCourtEmail::class.java to courtBookingRequestPrisonCourtEmail,
+    CourtBookingRequestPrisonNoCourtEmail::class.java to courtBookingRequestPrisonNoCourtEmail,
+    ProbationBookingRequestUserEmail::class.java to probationBookingRequestUser,
+    ProbationBookingRequestPrisonProbationTeamEmail::class.java to probationBookingRequestPrisonProbationTeamEmail,
+    ProbationBookingRequestPrisonNoProbationTeamEmail::class.java to probationBookingRequestPrisonNoProbationTeamEmail,
+    TransferredCourtBookingCourtEmail::class.java to transferCourtBookingCourt,
+    TransferredCourtBookingPrisonCourtEmail::class.java to transferCourtBookingPrisonCourtEmail,
+    TransferredCourtBookingPrisonNoCourtEmail::class.java to transferCourtBookingPrisonNoCourtEmail,
+    ReleasedCourtBookingCourtEmail::class.java to releaseCourtBookingCourt,
+    ReleasedCourtBookingPrisonCourtEmail::class.java to releaseCourtBookingPrisonCourtEmail,
+    ReleasedCourtBookingPrisonNoCourtEmail::class.java to releaseCourtBookingPrisonNoCourtEmail,
+  )
+
+  init {
+    require(EmailTemplates::class.constructors.single().parameters.size == emailTemplateMappings.size) {
```

Comment:
This ensures there is always a mapping for each template ID to the respective email.  This removes the need to check if an email is supported in the email service when sending.  As a result supporting new emails should never be missed.